gaussian-sub.py

```diff
@@ -115,7 +121,8 @@ def get_user_input():
         if allocation == '':
             print(textwrap.fill(textwrap.dedent("""\
                 ERROR: You must specify an allocation"""),100))
-            sys.exit()
+            #sys.exit()
+            allocation = 'hyak-stf'
```

Comment:
We were specifically asked by UW-IT not to have a default group, so remove this (see #3 )